package/MDAnalysis/lib/NeighborSearch.py

```diff
@@ -73,21 +65,20 @@ def search(self, atoms, radius, level='A'):
 
         Parameters
         ----------
-        atoms : AtomGroup, MDAnalysis.core.groups.Atom
-          list of atoms
+        atoms : AtomGroup, MDAnalysis.core.groups.AtomGroup
+          AtomGroup object
```

Comment:
Please add a Returns section that states what kind of groups are returned.

Please also add a versionchanged 2.0.0 stating the changed behavior.